infrastructure/php-container/src/.htaccess

```diff
@@ -19,21 +19,21 @@ DirectorySlash off
     RewriteRule ^graphql$ api/public/graphql [L]
 
     # Send admin requests to admin public folder; don't rewrite twice.
-    RewriteCond %{REQUEST_URI} !^admin/public/
-    RewriteRule ^admin(/(.*))?$ admin/public/$2 [L]
+    RewriteCond %{REQUEST_URI} !^frontend/admin/public/
```

Comment:
This line is no longer necessary, since after rewriting once it will no longer match the rewriteRule pattern

Reply:
Sorry, I haven't worked with apache in a few years, so a little rusty: Are you saying "this was never needed all along", or that "these dir changes have made it unnecessary"?